tools/sdk-ai-bots/azure-sdk-qa-bot-backend/handler/feedback_handler.go

```diff
@@ -15,11 +16,21 @@ func FeedBackHandler(c *gin.Context) {
 		return
 	}
 
-	service := feedback.NewFeedbackService()
-	if err := service.SaveFeedback(req); err != nil {
+	// Save feedback to Excel
+	feedbackService := feedback.NewFeedbackService()
+	if err := feedbackService.SaveFeedback(req); err != nil {
 		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
 		return
 	}
 
+	// Create GitHub sub-issue
+	go func() {
```

Comment:
better to file an issue only needed (the feedback is dislike, not need to start a thread every feedback to save resource